src/popup/locales/ru.json

```diff
@@ -231,8 +231,8 @@
       "backup": "backup",
       "yourSeedPhrase": "your seed phrase",
       "updateExtension": "New version available! Please update your extension.",
-      "updateAvailable": "New version in store available!",
-      "claim-name": "Claim your .chain name"
+      "updateAvailable": "New version in the store is available!",
+      "claim-name": "Register the name"
```

Comment:
Shouldn't it be `Register a name` instead?

Reply:
Yes, it should. Because no there no any current name before it's defined.